component/component/vshn_appcat_services.jsonnet

```diff
@@ -0,0 +1,213 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+
+local comp = import 'lib/appcat-compositions.libsonnet';
+local crossplane = import 'lib/crossplane.libsonnet';
+
+local common = import 'common.libsonnet';
+local prom = import 'prometheus.libsonnet';
+local xrds = import 'xrds.libsonnet';
+
+local slos = import 'slos.libsonnet';
+
+local inv = kap.inventory();
+local params = inv.parameters.appcat;
+
+local serviceNameLabelKey = 'appcat.vshn.io/servicename';
+local serviceNamespaceLabelKey = 'appcat.vshn.io/claim-namespace';
+
+local getServiceNamePlural(serviceName) =
+  local serviceNameLower = std.asciiLower(serviceName);
+  if std.endsWith(serviceName, 's') then
+    serviceNameLower
+  else
+    serviceNameLower + 's';
+
+local vshn_appcat_service(name) =
+  local isOpenshift = std.startsWith(inv.parameters.facts.distribution, 'openshift');
+  local securityContext = if isOpenshift then false else true;
```

Comment:
Isn't `isOpenshift` already a boolean? Can't you just do something like `!isOpenshift`?

Reply:
yeah makes sense. Fixed